lib/iiif/v3/presentation/annotation_page.rb

```diff
@@ -3,14 +3,27 @@ module V3
     module Presentation
       class AnnotationPage < IIIF::V3::AbstractResource
 
-        TYPE = 'AnnotationPage'
+        TYPE = 'AnnotationPage'.freeze
 
         def required_keys
           super + %w{ id }
         end
 
+        def prohibited_keys
+          super + CONTENT_RESOURCE_PROPERTIES +
+            %w{ first last total nav_date viewing_direction start_canvas content_annotations }
+        end
```

Comment:
Looking through the 3.0 (alpha) spec, I couldn't find e.g. `content_annotations`. I guess I must be missing something, otherwise you wouldn't have added this?